README.md

```diff
@@ -1 +1,20 @@
-# gluster-mixins
+# Prometheus Monitoring Mixin for Gluster
+
+A set of Grafana dashboards and Prometheus alerts for Gluster.
+
+The scope of this project is to provide Gluster specific Grafana dashboard configs and Prometheus rule files using Prometheus Mixins. 
+
+## Prerequisites
+1. Jsonnet ([How to Install Jsonnet?](https://github.com/google/jsonnet#building-jsonnet))
+
+## How to use?
+1. `$ git clone https://github.com/gluster/gluster-mixins.git`
```

Comment:
These are not the steps. can you remove the marking saying 1, 2 , ...

Reply:
ack.